tempesta_fw/sock.c

```diff
@@ -405,6 +402,15 @@ ss_do_close(struct sock *sk)
 	}
 }
 
+#define __ss_sk_close(sk, lock, unlock, drop)				\
```

Comment:
It's hard to read code like `__ss_sk_close(sk, 1, 1, 1)` - what the numbers are? It's hard to remember. Let's introduce defines like `__ss_sk_close_locked()` or `__ss_sk_close_drop_locked()`.